src/k8s/pkg/utils/file.go

```diff
@@ -16,20 +16,32 @@ import (
 // ParseArgumentLine parses a command-line argument from a single line.
 // The returned key includes any dash prefixes.
 func ParseArgumentLine(line string) (key string, value string) {
-	line = strings.TrimSpace(line)
+	line = strings.TrimSpace(line) // Trim leading and trailing white spaces
 
-	// parse "--argument value" and "--argument=value" variants
```

Comment:
@eaudetcobello i found this comment line SUPER important.  I was trying to figure out why we were looking for " " or "="

qq:  is `--argument         value` legal?

Reply:
I would think so. There's two test cases for that:

```
		{line: "--key    =", key: "--key", value: "="},
		{line: "--key    = a value=", key: "--key", value: "= a value="},
```